.github/workflows/docker.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Setup Docker QEMU
         uses: docker/setup-qemu-action@v1.2.0
       - name: Setup Docker Buildx
-        uses: docker/setup-buildx-action@v1.6.0
+        uses: docker/setup-buildx-action@v1
```

Comment:
Do we really want to drop the sub-version part?

Reply:
I don't see why not :shrug: 
I wouldn't expect anything to break in a sub-version change.